docs/workflows/standalone/dorado_basecalling.md

```diff
@@ -0,0 +1,111 @@
+|---|---|---|---|---|
+| [Standalone](../../workflows_overview/workflows_type.md/#standalone) | [Any Taxa](../../workflows_overview/workflows_kingdom.md/#any-taxa) | v2.2.1 | Yes | Sample-level |
+
+## Dorado_Basecalling_PHB
+
+The Dorado Basecalling workflow is used to convert Oxford Nanopore `POD5` sequencing files into `FASTQ` format by utilizing a GPU-accelerated environment. This workflow is ideal for high-throughput applications where fast and accurate basecalling is essential. The workflow will upload fastq files to a user designated terra table for downstream analysis.
+
+### Model Type Selection
+
+Users can configure the basecalling model by setting the `dorado_model` input parameter:
+
+Default Model: "sup" (super accuracy) is used unless overridden by the user.
+Manual Model Input: Users can specify the full path or name of a specific model (e.g., dna_r10.4.1_e8.2_400bps_hac@v4.2.0).
+Automatic Detection: When set to sup, hac, or fast, Dorado will automatically select the appropriate model version if available.
+
+- **Model Type (sup):** (Super Accuracy) The most accurate model, recommended for critical applications requiring the highest basecall accuracy. It is the slowest of the three model types.
+- **Model Type (hac):** (High Accuracy) A balance between speed and accuracy, recommended for most users. Provides accurate results faster than `sup` but less accurate than `sup`.
+- **Model Type (fast):** (Fast Model) The fastest model, recommended when speed is prioritized over accuracy, such as for initial analyses or non-critical applications.
+
+### Example Manual Models:
+
+- `dna_r10.4.1_e8.2_400bps_sup@v4.2.0`
+- `dna_r10.4.1_e8.2_400bps_hac@v4.2.0`
+- `dna_r10.4.1_e8.2_400bps_fast@v4.2.0`
+
+??? Info "Supported Kit Names"
+
+    Ensure you use an accepted kit name in the `kit_name` parameter. The following kit names are supported in Dorado:
+
+    ```
+    EXP-NBD103, EXP-NBD104, EXP-NBD114, EXP-NBD114-24, EXP-NBD196, EXP-PBC001, EXP-PBC096, 
+    SQK-16S024, SQK-16S114-24, SQK-LWB001, SQK-MLK111-96-XL, SQK-MLK114-96-XL, SQK-NBD111-24, 
+    SQK-NBD111-96, SQK-NBD114-24, SQK-NBD114-96, SQK-PBK004, SQK-PCB109, SQK-PCB110, SQK-PCB111-24, 
+    SQK-PCB114-24, SQK-RAB201, SQK-RAB204, SQK-RBK001, SQK-RBK004, SQK-RBK110-96, SQK-RBK111-24, 
+    SQK-RBK111-96, SQK-RBK114-24, SQK-RBK114-96, SQK-RLB001, SQK-RPB004, SQK-RPB114-24, 
+    TWIST-16-UDI, TWIST-96A-UDI, VSK-PTC001, VSK-VMK001, VSK-VMK004, VSK-VPS001
+    ```
+
+    Select from these options to avoid input errors.
+
+### Inputs
+
+| **Terra Task Name** | **Variable** | **Type** | **Description** | **Default Value** | **Terra Status** |
+|---|---|---|---|---|---|
+| dorado_basecalling_workflow | **input_files** | Array[File] | Array of `POD5` files for basecalling | None | Required |
+| dorado_basecalling_workflow | **dorado_model** | String | Model accuracy or full model name (default: 'sup') | "sup" | Optional |
```

Comment:
I think this line is slightly inaccurate? `sup`, `hac`, and `fast` are **model speeds** , rather than the actual model itself: https://github.com/nanoporetech/dorado?tab=readme-ov-file#model-selection-foreword

Reply:
not a big deal, but it might be good to stay consistent with ONT's language